lib/chef/resource/sysctl.rb

```diff
@@ -45,6 +45,10 @@ class Sysctl < Chef::Resource
                coerce: proc { |v| coerce_value(v) },
                required: true
 
+      property :comment, [Array, String],
+               description: "An optional comment to prepend the resource setting.",
```

Comment:
We generate our docs from these comments so you'll want to briefly explain that multi-line comments can be accomplished by using an array of strings where each line is a value in the array.

Reply:
OK, have updated and pushed.  Thanks.

Reply:
hmmm, that didn't work. All tests pass locally. moving it back to a single line to test.